aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -218,5 +219,22 @@ bool quantized_equal(const Tensor& self, const Tensor& other) {
   return 0 == memcmp(self_data, other_data, self.numel() * self.element_size());
 }
 
+/* Calculate the quantization params for the activation tensor */
+std::tuple<double, int64_t> _choose_qparams(const Tensor& self, bool reduce_range) {
```

Comment:
we'll need to have another function for per channel quantization as well until we expose Quantizer to python, maybe rename this to `_choose_qparams_per_tensor` now?

Reply:
Do we need per-channel quantization for activations though?

Reply:
probably not, but this is still a `_choose_qparams` only for per tensor quantization, I feel it's better to be more explicit.